functions/v2/http-logging/test/index.test.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const assert = require('assert');
+const sinon = require('sinon');
+const { getFunction } = require('@google-cloud/functions-framework/testing');
+
+describe('structured logging: functions http', () => {
+	let req, res, spy;
+
+	before(async () => {
```

Comment:
Since the goal of the sample is to take advantage of the new stdout log delivery of the library, it would be good to make this a more comprehensive test and double-check we're using that as expected by testing captured stdout instead of intercepting the library call.

Reply:
yes, let's remove the mocking and just test the stdout. A similar example can be found: https://github.com/googleapis/nodejs-dlp/blob/main/samples/system-test/quickstart.test.js